tests/unit/ReportUtilsGetIconsTest.ts

```diff
@@ -0,0 +1,535 @@
+            chatType: CONST.REPORT.CHAT_TYPE.POLICY_ANNOUNCE,
+            policyID: '1',
+        };
+        const policy = LHNTestUtils.getFakePolicy('1');
+
+        // Verify report type conditions
+        expect(isAnnounceRoom(report)).toBe(true);
+        expect(isChatRoom(report)).toBe(true);
+
+        const icons = getIcons(report, FAKE_PERSONAL_DETAILS, null, '', -1, policy);
+        expect(icons).toHaveLength(1);
+        expect(icons.at(0)?.type).toBe(CONST.ICON_TYPE_WORKSPACE);
+    });
+
+    it('should return the correct icons for an invoice room with individual receiver', () => {
+        const report: Report = {
+            ...LHNTestUtils.getFakeReport([], 0, true),
+            chatType: CONST.REPORT.CHAT_TYPE.INVOICE,
+            policyID: '1',
+            invoiceReceiver: {
+                type: CONST.REPORT.INVOICE_RECEIVER_TYPE.INDIVIDUAL,
+                accountID: 2,
+            },
+        };
+        const policy = LHNTestUtils.getFakePolicy('1');
+
+        // Verify report type conditions
+        expect(isInvoiceRoom(report)).toBe(true);
+        expect(isIndividualInvoiceRoom(report)).toBe(true);
+
+        const icons = getIcons(report, FAKE_PERSONAL_DETAILS, null, '', -1, policy);
+        expect(icons).toHaveLength(2); // Workspace + individual receiver
+        expect(icons.at(0)?.type).toBe(CONST.ICON_TYPE_WORKSPACE);
+        expect(icons.at(1)?.type).toBe(CONST.ICON_TYPE_AVATAR);
+    });
+
+    it('should return the correct icons for an invoice room with business receiver', () => {
+        const receiverPolicy = LHNTestUtils.getFakePolicy('2', 'Receiver-Policy');
+        const report: Report = {
+            ...LHNTestUtils.getFakeReport([], 0, true),
+            chatType: CONST.REPORT.CHAT_TYPE.INVOICE,
+            policyID: '1',
+            invoiceReceiver: {
+                type: CONST.REPORT.INVOICE_RECEIVER_TYPE.BUSINESS,
+                policyID: '2',
+            },
+        };
+        const policy = LHNTestUtils.getFakePolicy('1');
+        const icons = getIcons(report, FAKE_PERSONAL_DETAILS, null, '', -1, policy, receiverPolicy);
+        expect(icons).toHaveLength(2); // Workspace + receiver workspace
+        expect(icons.at(0)?.type).toBe(CONST.ICON_TYPE_WORKSPACE);
+        expect(icons.at(1)?.type).toBe(CONST.ICON_TYPE_WORKSPACE);
+        expect(icons.at(1)?.name).toBe('Receiver-Policy');
+    });
+
+    it('should return the correct icons for a multi-transaction IOU report where current user is not manager', () => {
+        const report: Report = {
+            ...LHNTestUtils.getFakeReport([1, 2], 0, true),
+            reportID: 'multiTxn',
+            chatReportID: 'chatReport',
+            type: CONST.REPORT.TYPE.IOU,
+            ownerAccountID: 1,
+            managerID: 2, // Different from current user (5)
+            // eslint-disable-next-line @typescript-eslint/naming-convention
+            participants: {
```

Comment:
Haven't checked out the branch to double check this, but what is wrong with the word `participants` such that we need to add a `eslint-disable-next-line @typescript-eslint/naming-convention`?